hpedockerplugin/hpe/hpe_3par_common.py

```diff
@@ -37,6 +37,7 @@
 DEDUP_API_VERSION = 30201120
 FLASH_CACHE_API_VERSION = 30201200
 COMPRESSION_API_VERSION = 30301215
+TIME_OUT = 5
```

Comment:
This value needs to be used for all the WSAPI calls except for the initial handshake where the plugin comes to know if the supplied URL is good or not. Hence, let's keep the value of this to be 20 or 30 seconds.
For the initial handshake, that happens in volume_manager.initialize_driver(), please pass a value of 5 seconds in common.do_setup() call, so that user would come to know quickly in case the URL is bad.